pkg/store/store.go

```diff
@@ -29,6 +29,20 @@ func LimaDir() (string, error) {
 	return dir, nil
 }
 
+// LimaConfigDir returns the path of the config directory, $LIMA_HOME/_config.
+// It will create the directory if it doesn't yet exist.
```

Comment:
I’m not sure this function should make the dir.

Reply:
Yeah, it is inconsistent with what `LimaDir` does, but I was thinking about letting that one create the directory too. Would that simplify the calling code?